ddtrace/profiling/exporter/http.py

```diff
@@ -147,15 +144,7 @@ def _get_tags(service):
 
         user_tags = os.getenv("DD_PROFILING_TAGS")
```

Comment:
I'm also wondering if we should add `DD_TAGS` here, no?

Reply:
probably, share the same tags between APM and profiling?